sdk/eventhub/azure-eventhub/azure/eventhub/_eventprocessor/event_processor.py

```diff
@@ -308,6 +318,24 @@ def _close_consumer(self, partition_id, consumer, reason):
 
         self._ownership_manager.release_ownership(partition_id)
 
+    def _do_receive(self, partition_id, consumer):
```

Comment:
Thanks for breaking out some of these sections into separate functions - helps make the code easier to read!
Though adding type hints and a simple docstring would make it even awesomer. :)